styles/prosilver/template/event/viewtopic_body_footer_before.html

```diff
@@ -0,0 +1,6 @@
+<!-- IF not $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS -->
+	<!-- INCLUDEJS js/functions.js -->
+	<!-- INCLUDEJS js/onload.js -->
+	<!-- INCLUDEJS js/scoring_topic.js -->
+    <!-- DEFINE $INCLUDED_WOLFSBLVTADVANCEDPOLLSJS = true -->
+<!-- ENDIF -->
```

Comment:
Why have you moved the include to the footer?
INCLUDEJS will always include the JS files in the footer, that's the cool thing about this function. So there is no need to seperate it. The if-construct prevents from adding several times in the loop, so I don't feel the need to create another event.
I am not quite sure how expensive events are while loading the template (for runtime, I mean), but why an extra file here? :D